modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,26 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: <project_name> # <2>
+  name: <service-monitor-name> <1>
+  namespace: <namespace> <2>
```

Comment:
Use the `{my-product-namespace}` attribute. We just fixed all the codebase in that direction. See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/858